solutions/pytorch_tests/config.json

```diff
@@ -0,0 +1,16 @@
+{
+    // OpenEnclave specific values
+
+    // Whether we are running myst+OE+app in debug mode
+    "Debug": 1,
+    "ProductID": 1,
+    "SecurityVersion": 1,
+
+    // Mystikos specific values
+
+    // The heap size of the user application. Increase this setting if your app experienced OOM.
+    "MemorySize": "8g",
```

Comment:
This is really troublesome. Why does PyTorch take so much memory, even with ext2fs?

Reply:
If I change this value to 2g, 6 additional test cases failed. I will split them.